lib/ftp/doc/src/ftp.xml

```diff
@@ -574,6 +574,9 @@
 	The list <c>tls_options()</c> can be empty. The function 
 	<seemfa marker="ssl:ssl#connect/3"><c>ssl:connect/3</c></seemfa> 
 	is used for securing both the control connection and the data sessions.
+	By supplying <c>{reuse_sessions,save}</c> the client will re-use the
```

Comment:
I think it would be less confusing having a ftp option here rather than making it look like an ssl option that does not behave as that ssl option does. Maybe {reuse_ctrl_session, boolean()}